### PR TITLE
Clean up some tests

### PR DESCRIPTION
**Pull request by dougbu**
- avoid mocking `HttpContext`
- make `DefaultAntiforgeryTest` mock token generators consistently

(Found I was cleaning up tests in other work. Decided to separate concerns.)

### CODE REVIEW

File: test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultAntiforgeryTokenStoreTest.cs [hunk 1]
```diff
@@ -157,7 +138,7 @@ public void GetCookieToken_CookieIsValid_ReturnsToken()
                 tokenSerializer: mockSerializer.Object);
 
             // Act
-            AntiforgeryToken retVal = tokenStore.GetCookieToken(mockHttpContext);
+            AntiforgeryToken retVal = tokenStore.GetCookieToken(httpContext);
```
Comment (rynowak) [inline test/Microsoft.AspNetCore.Antiforgery.Test/Internal/DefaultAntiforgeryTokenStoreTest.cs]: var + something other than `retVal`

**Comment by rynowak** [comment]
:shipit:

**Comment by dougbu** [comment]
c91f0ee

**Pull request closed without merging**